libs/features/components/cards/src/lib/cards/elewa-key-figure-card/elewa-key-figure-card.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="container">
+    <div class="cardSection">
+        <div class="figure" isImage>
+            <img src={{figure}} alt="">
+            <p class="desc">{{description}}</p>
+        </div>
+    </div>    
+</div>
```

Comment:
use structural directives here... i.e. *ngIf

Reply:
@IanOdhiambo9 please review the current changes